src/features/rewards/disabledPanel/index.tsx

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import * as React from 'react'
+import {
+  StyledWrapper,
+  StyledTitle,
+  StyledOff,
+  StyledText,
+  StyledLink,
+  StyledTitleWrapper,
+  StyledTextWrapper
+} from './style'
+
+import { getLocale } from '../../../helpers'
+
+export interface Props {
+  id?: string
```

Comment:
I think we are using test-id or something now? could be mistaken

Reply:
I don't think we are explicitly supposed to be using something but yeah this should probably be testId instead of Id

Reply:
added